### PR TITLE
fix: socket message throw

### LINKED ISSUE #456: Local crashes when restoring: "connection doesn't exist" for web sockets
This error occurs when i reboot the local server using web sockets. Of course the connection doesn't exist - we should not crash!

```
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: Websocket for connection 200d0388-ff8b-42c2-a913-acfcc7339ec1 does not exist.
```

### PR DESCRIPTION
**Pull request by thantos**
Fixes #456 

* Fix bug where an error in the message handler would be uncaught in and crash the server.
* Local command/http tests were always calling the cloud url, which... is wrong. Fixed that.

### CODE REVIEW

File: apps/tests/aws-runtime/scripts/start-local.js [hunk 1]
```diff
@@ -27,12 +27,15 @@ const firstProcess = spawn("node", [eventual, "local"], {
...
-    const secondProcess = spawn("pnpm", ["test:local"], {
-      env: envVars,
-    });
+    const secondProcess = spawn(
+      "pnpm",
+      ["test:local", ...process.argv.slice(2)],
+      {
+        env: envVars,
+      }
+    );
```
Comment (sam-goodwin) [inline apps/tests/aws-runtime/scripts/start-local.js:38]: Why second process?

File: apps/tests/aws-runtime/test/tester.test.ts [hunk 2]
```diff
@@ -484,6 +484,14 @@ function setupWS(executionId: string, ws: WebSocket) {
       reject(err);
     });
     ws.on("message", (data) => {
+      // test: invalid call, should not crash the server
```
Comment (sam-goodwin) [inline apps/tests/aws-runtime/test/tester.test.ts:487]: Has this been tested?

**Review by sam-goodwin — APPROVED** [review]

File: apps/tests/aws-runtime/scripts/start-local.js (hunk 1 above)
Comment (thantos) [reply apps/tests/aws-runtime/scripts/start-local.js:38]: I wanted the args into test so I could filter the tests down to a specific test.

File: apps/tests/aws-runtime/test/tester.test.ts (hunk 2 above)
Comment (thantos) [reply apps/tests/aws-runtime/test/tester.test.ts:487]: yes

Comment (thantos) [reply apps/tests/aws-runtime/test/tester.test.ts:487]: Though it seems to be failing the runtime tests....

**Pull request merged**